himanshu/postgresAssignment/src/index.ts

```diff
@@ -0,0 +1,21 @@
+const Koa = require('koa');
+
+const app = new Koa();
+const json = require('koa-json');
+const bodyparser = require('koa-bodyparser');
+
+const port = 3000;
+const combineRouters = require('koa-combine-routers');
+
+app.use(json());
+app.use(bodyparser());
+
+const router = combineRouters(require('./routes/studentRoutes.ts'), require('./routes/resultRoutes.ts'));
+
+app.use(router());
+
+const server = app.listen(port, () => {
```

Comment:
maybe export app and initialise server in separate file, this help in writing testcaes